WordPressKit/HTTPRequestBuilder.swift

```diff
@@ -215,7 +215,7 @@ final class HTTPRequestBuilder {
         request.setValue("text/xml", forHTTPHeaderField: "Content-Type")
         let encoder = WPXMLRPCEncoder(method: xmlrpcRequest.method, andParameters: xmlrpcRequest.parameters)
         if forceWriteToFile {
-            let fileName = "\(ProcessInfo.processInfo.globallyUniqueString)_file.xmlrpc"
+            let fileName = "\(UUID().uuidString).xmlrpc"
```

Comment:
[Naming temp files using PID](https://github.com/wordpress-mobile/WordPressKit-iOS/blob/e7d2c04c1be6c602c2f285643f6e1e19f935a8d7/WordPressKit/WordPressOrgXMLRPCApi.swift#L257-L261) can be problematic if there are concurrent uploads.

Reply:
I can't verify this assumption though, maybe Foundation API is safe enough to avoid concurrent writes to the same file. Nonetheless, I don't think we should reuse the same file, given temp files will now be removed after request completes.

Reply:
Using a unique name sounds like a good idea regardless 👍

Reply:
Actually, I misread the code, the API is "globally unique string", not "PID". 🤦‍♂️ 

The original code is an alternative to `UUID`, which means this code change is kind of meaningless. 😄